app/src/components/domain/RiskImminentEvents/Gdacs/index.tsx

```diff
@@ -54,6 +99,12 @@ function Gdacs(props: Props) {
         variant,
     } = props;
 
+    const [
+        activeLayersMapping,
+        setActiveLayersMapping,
+    ] = useState<Record<number, boolean>>(defaultLayersValue);
+    const [layers, setLayers] = useState<Record<number, boolean>>(defaultLayersValue);
```

Comment:
What is the different between these 2 layers?

Reply:
`activeLayersMapping` persists the state of layers across interactions.
`layers` handles the temporary, interactive behavior of showing or hiding layers.